server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1726,13 +1728,13 @@ public void importDataFiles(long tid, Map<ReferencedTabletFile,DataFileInfo> fil
 
     // Clients timeout and will think that this operation failed.
     // Don't do it if we spent too long waiting for the lock
-    long now = System.nanoTime();
+    NanoTime now = NanoTime.now();
     synchronized (this) {
       if (isClosed()) {
         throw new IOException("tablet " + extent + " is closed");
       }
 
-      long rpcTimeoutNanos = TimeUnit.MILLISECONDS.toNanos(
+      Duration rpcTimeout = Duration.ofNanos(
```

Comment:
The unit seem off here.

```suggestion
      Duration rpcTimeout = Duration.ofMillis(
```